ybacklight

```diff
@@ -79,9 +79,9 @@ sub set_percentage {
         $pct = 100;
     }
     my $bri_max = get_numeric_attribute($driver, "max_brightness");
-    my $bri_new = int($bri_max/100*$pct); 
+    my $bri_new = ($bri_max/100*$pct);
     say "Setting brightness to $bri_new/$bri_max ($pct%)" if $verbose;
-    set_attribute($driver, "brightness", $bri_new);
+    set_attribute($driver, "brightness", int($bri_new));
```

Comment:
Is this change now needed at all? $bri_new is only used in the say above, which now potentially prints a fractional value that it’s not going to set.